test/mainline_test.go

```diff
@@ -347,6 +350,33 @@ func installResourceCRD(c client.Client, mgr manager.Manager, ctx context.Contex
 	err := c.Create(context.Background(), instance)
 	Expect(err).NotTo(HaveOccurred())
 
+	ns := &corev1.Namespace{
```

Comment:
I don't think I understand this change. This function is meant to be a helper for installing the `Installation` CR - why does it also need to create the DaemonSet now?

Reply:
I think we need to revert this - it's a blanket change for all of the existing tests that AFAICT isn't really appropriate.

If you want to run an FV for this case, we need to have a separate test case targeted at it.